src/storage/mod.rs

```diff
@@ -292,3 +305,28 @@ fn should_detect_zeroes() {
   let nums = vec![1; 10];
   assert!(not_zeroes(&nums));
 }
+
+#[cfg(test)]
+mod test {
+  use super::*;
+  use crypto::generate_keypair;
+  use sha2::Sha512;
+  use ed25519_dalek::{Keypair};
```

Comment:
This seemed right to avoid an unused import, let me know if there's a better way!